pkg/apis/sparkoperator.k8s.io/v1beta2/types.go

```diff
@@ -264,6 +264,8 @@ type SparkApplicationSpec struct {
 	// BatchSchedulerOptions provides fine-grained control on how to batch scheduling.
 	// Optional.
 	BatchSchedulerOptions *BatchSchedulerConfiguration `json:"batchSchedulerOptions,omitempty"`
+	// DeleteOnTermination specify whether executor pods should be deleted in case of failure or normal termination
+	DeleteOnTermination *bool `json:"deleteOnTermination,omitempty"`
```

Comment:
I suggest move this field to go under `Executor`.

Reply:
Makes sense. I will make the change.